public/js/sserve/controllers.js

```diff
@@ -1,856 +1,907 @@
 'use strict';
-/* global projectMigrationApp, angular, _, moment, $ */
+/* global projectMigrationApp, angular, _, moment, $, transformMigrations, prepareReport, transformMigrated */
```

Comment:
These are global functions. The commented line is actually read by jsHint - adding these globals  avoids having jsHint  complain about the globals.